src/generator/AutoRest.Java.Azure/Templates/SimulatedPageMethodTemplate.cshtml

```diff
@@ -23,18 +24,19 @@
 }
 @if (Model.ReturnType.Body != null)
 {
-@: * @@return the @Model.ReturnTypeResponseName.EscapeXmlComment() object if successful.
+@: * @@return the PagedList<@Model.ReturnTypeJv.SequenceElementTypeString> object if successful.
```

Comment:
Results in to making all list function return PagedList
https://github.com/Azure/azure-sdk-for-java/pull/1507/files#diff-4e9ba83c71e8f9426206d540d5c7a1c8R633